### PR TITLE
Exclude compiler service benchmark from VMR build when not building tests

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Review by vzarytovskii — APPROVED** [review]

File: tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/FSharp.Compiler.Benchmarks.fsproj [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
+    <ExcludeFromDotNetBuild Condition="'$(DotNetBuildTests)' != 'true'">true</ExcludeFromDotNetBuild>Z
```
Comment (ViktorHofer) [inline tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/FSharp.Compiler.Benchmarks.fsproj:6]: Remove the Z at the end. Note that fsharp is on Arcade 8 so you might need to use a different property here.

**Comment by psfinaki** [comment]
Should any other benchmark projects be also removed from that build then?

File: tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/FSharp.Compiler.Benchmarks.fsproj (hunk 1 above)
Comment (mmitche) [reply tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/FSharp.Compiler.Benchmarks.fsproj]: Arcade 8 isn't used in context of building this in the VMR (and this is excluded elsewhere in SB).

**Comment by mmitche** [comment]
> Should any other benchmark projects be also removed from that build then?

I was only removing the projects that were blocking the build and causing issues. I think this one is an issue because other test projects weren't being built, and this project references them.

**Review by ViktorHofer — APPROVED** [review]

**Review by psfinaki — APPROVED** [review]

**Pull request merged**